rasa_core/train.py

```diff
@@ -47,21 +55,37 @@ def create_argument_parser():
             default=20,
             help="number of training samples to put into one training batch")
     parser.add_argument(
+            '--online',
```

Comment:
if there's a short option for --nlu and --verbose, should there not also be one for --online? e.g -o ?

Reply:
`-o` is already used for `output` in other scripts of Rasa, so to keep it consistent I'd avoid a short letter